python-traceback2 (1.4.0-5) UNRELEASED; urgency=medium

  * Bumped debhelper compat version to 10

 -- Ondřej Nový <onovy@debian.org>  Thu, 24 Nov 2016 00:12:03 +0100

python-traceback2 (1.4.0-4) unstable; urgency=medium

  * Fixed VCS URLs (https).
  * d/rules: Changed UPSTREAM_GIT protocol to https
  * d/s/options: extend-diff-ignore of .gitreview
  * d/control: Using OpenStack's Gerrit as VCS URLs.

 -- Ondřej Nový <novy@ondrej.org>  Sun, 28 Feb 2016 15:52:22 +0100

python-traceback2 (1.4.0-3) unstable; urgency=medium

  * Fixed FTBFS by not running unit tests: there's no way to do it the way
    upstream organized sources and dependencies (Closes: #799163).

 -- Thomas Goirand <zigo@debian.org>  Mon, 21 Sep 2015 08:48:26 +0000

python-traceback2 (1.4.0-2) unstable; urgency=medium

  * Applied Ubuntu patch to run unit tests at build time (Closes: #795891).

 -- Thomas Goirand <zigo@debian.org>  Mon, 14 Sep 2015 11:14:16 +0000

python-traceback2 (1.4.0-1) unstable; urgency=medium

  * Initial release. (Closes: #791791)

 -- Thomas Goirand <zigo@debian.org>  Tue, 21 Apr 2015 09:25:58 +0200
